src/tools/printjob.c

```diff
@@ -547,6 +547,9 @@ char *argv[];
 		if (fp < 0) {
 
 #ifdef PRINTJOBSVR
+#ifdef WIN32
```

Comment:
I think you need to move the winsock_init in the case of PRINTJOBSVR to before the loop (and so would the cleanup be after the loop finishes entirely, else we will do init and cleanup for every possible iteration.